### DISCUSSION TITLE
What is Developer Program Member?

### DISCUSSION THREAD

**Discussion by brilliantAlmaz** [community/community#35180] [Other Feature Feedback, Questions, & Ideas] [answered]
I have seen a couple of Github profiles having "Developer Program Member", can anyone explain what is the essence of this and how it works?

**Comment by ghost** [comment, marked as answer]
[GitHub Developer Program](https://docs.github.com/en/developers/overview/github-developer-program) - This is a program that developers join who build tools that integrate with GitHub.

This is for developers registered to the [GitHub Developer Program](https://developer.github.com/program/)
That allows for:
> **Staying in the know:**
> Be the first to know about API changes and try out new features before they launch.
> 
> **Scratching an itch**
> Build your own tools that seamlessly integrate with the place you push code every day.
> 
> **Taking on the enterprise:**
> Obtain developer licenses to build and test your application against GitHub Enterprise.

**Comment by pn-1705** [comment]
ib

**Comment by pn-1705** [comment]
ib
